app/assets/javascripts/tylium/modules/issues/tag-input.js.coffee

```diff
@@ -5,3 +5,29 @@ document.addEventListener "turbolinks:load", ->
     $span = $("#issues_editor .dropdown-toggle span.tag")
     $span.html($target.html())
     $span.css("color", $target.css("color"))
+
+    # Trigger local auto save and store selected tag value in local cache
+    $form = $target.parents('form[data-behavior~=local-auto-save]')
+    if $form
+      $('#issue_tag_list').trigger('textchange')
+
+  # Not really fond of putting this piece of code here, but I'm not sure where else to put it.
```

Comment:
This is a comment that's meant for something like GitHub where we can have a discussion about it. That said, there's nothing wrong with splitting this into a new file. `local_tag_auto_save.js`?

Reply:
Thanks for the suggestion, i've refactored it into a new file.